config/locales/en.yml

```diff
@@ -635,3 +635,5 @@ Please follow the instructions there to make your enterprise visible on the Open
   price_graph: "Price graph"
   included_tax: "Included tax"
   remove_tax: "Remove tax"
+  current_version: "Current Version"
```

Comment:
Perhaps a matter of taste, but I'd prefer _Current version_ (with a lowercase "v") here. As long is it isn't a typo :)